documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -456,32 +468,43 @@ public class Essay {
     ...
 
     @Id
-    @DocumentId
+    @GeneratedValue
     public Long getId() { return id; }
 
-    @Field(name="Abstract", store=Store.YES)
-    public String getSummary() { return summary; }
-
-    @Lob
-    @Field
-    public String getText() { return text; }
-
-    @Field
-    @NumericField(precisionStep = 6)
-    public float getGrade() { return grade; }
+    ...
 }
 ----
 ====
 
-<<example-annotated-entity>> defines an index with four fields: `id`, `Abstract`, `text` and
-`grade`. Note that by default the field name is de-capitalized, following the JavaBean
-specification. The `grade` field is annotated as numeric with a slightly larger `precisionStep` than
-the default.
+If you don't use JPA,
```

Comment:
We had set some style "house rules":
 - any period should be on a single line, with exceptions of "within reason" to break very long ones.
 - we don't like the style of "it is recommended". Better to use "we suggest"

I see it's gradually being forgotten as I'm getting non-picky so today I'll be extra demanding as you might not even ever have heard of these :)